netty/src/main/java/io/grpc/netty/GrpcHttp2OutboundHeaders.java

```diff
@@ -81,6 +81,21 @@ public CharSequence status() {
     return new Itr();
   }
 
+  /**
+   * Returns the first value whose key matches the input.
+   */
+  @Override
```

Comment:
im reluctant to add this method.  It really shouldn't exist since it isnt very efficient.  Can you just dump all the headers into a DefaultHttp2Headers if you want to check if they are present?